### PR TITLE
fix/vrh-loading-vehicle-status

### PR DESCRIPTION
**Pull request by bartverhaar**
Make sure vehicle-status is loaded afterr vrhAGS service is initialized

### CODE REVIEW

File: public/js/safetymaps/modules/incidents/VehicleIncidentsController.js [hunk 1]
```diff
@@ -116,7 +116,9 @@ function VehicleIncidentsController(options, featureSelector) {
     if(me.options.incidentSource === "VrhAGS" || me.options.incidentSourceFallback === "VrhAGS") {
         // Initialize AGS service
         me.service = new AGSIncidentService(me.options.apiPath + "vrhAGS", me.options.apiPath + "vrhAGSEenheden");
-        me.initializeService();
+        me.initializeService(function() {
+            me.options.showStatus && me.updateStatus();
```
Comment (matthijsln) [inline public/js/safetymaps/modules/incidents/VehicleIncidentsController.js]: Why is this necessary, `showStatusVrhAGS` sets a timeout in the `always()` for the `service.getVoertuigStatus()` promise so it is retried when it is initialized? If needed anyway use a Promise instead of a callback.

**Review by matthijsln — CHANGES_REQUESTED** [review]

File: public/js/safetymaps/modules/incidents/VehicleIncidentsController.js (hunk 1 above)
Comment (bartverhaar) [reply public/js/safetymaps/modules/incidents/VehicleIncidentsController.js]: @matthijsln see commit 7588ebf

**Review by matthijsln — APPROVED** [review]
